Makefile

```diff
@@ -209,9 +211,9 @@ verify: .init .generate_files verify-client-gen
 	    | grep -v v1alpha1/defaults.go); \
 	  do \
 	    golint --set_exit_status $$i; \
-	    go vet $$i; \
 	  done'
 	@#
+	$(DOCKER_CMD) go vet $(NON_VENDOR_DIRS)
```

Comment:
It probably would. I'll check. `go vet` specifically was in my personal queue of 'annoyances'.